src/openms/include/OpenMS/QC/Contaminants.h

```diff
@@ -39,24 +39,29 @@
 #include <OpenMS/METADATA/PeptideIdentification.h>
 #include <algorithm>
 #include <bits/stdc++.h>
+#include <OpenMS/QC/QCBase.h>
 
 
 namespace OpenMS
 {
-  class OPENMS_DLLAPI Contaminants
+  class OPENMS_DLLAPI Contaminants:
+      QCBase
   {
   public:
-    void compute(FeatureMap& features, const std::vector<FASTAFile::FASTAEntry>& contaminants);
-    const std::vector<std::pair<double, double>>& getResults();
     struct resultsData
     {
       double assigned_contaminants_ratio; //(# contaminants in assigned/ #peptides in assigned)
       double unassigned_contaminants_ratio; //(# contaminants in unassigned/ #peptides in unassigned)
       double all_contaminants_ratio; //(# all contaminants/ #peptides in all)
       double assigned_contaminants_intensity; //(intensity of contaminants in assigned/ intensity of peptides in assigned)
     };
+    Contaminants() = default;
+    virtual ~Contaminants() = default;
+    void compute(FeatureMap& features, const std::vector<FASTAFile::FASTAEntry>& contaminants);
```

Comment:
add documentation for all public methods, see https://github.com/cbielow/OpenMS/pull/42/files